lib/core/bufferFactory.cpp

```diff
@@ -103,7 +103,7 @@ struct Buffer* bufferFactory::new_buffer(const string& type_name, const string&
         auto layout = VisFrameView::calculate_buffer_layout(num_elements, num_prod, num_ev);
         uint32_t frame_size = layout.first;
 
-        INFO_NON_OO("Creating visBuffer named {:s} with {:d} frames, frame size of {:d} and "
+        INFO_NON_OO("Creating VisFrameView named {:s} with {:d} frames, frame size of {:d} and "
```

Comment:
```suggestion
        INFO_NON_OO("Creating VisBuffer named {:s} with {:d} frames, frame size of {:d} and "
```
I think this should still be `VisBuffer`, the frame view is just a useful abstraction on the buffer.